README.md

```diff
@@ -749,6 +783,15 @@ log-targets:
     # service name with a minus (e.g. '-svc1') to remove a previously added
     # service. '-all' will remove all services.
     services: [<service names>]
+
+    # (Optional) A list of key/value pairs defining labels which should be set
+    # on the outgoing logs. These labels are added to (protocol-dependent)
+    # default labels (e.g. 'pebble_service' for Loki).
+    # The label values may contain $ENVIRONMENT_VARIABLES, which will be
+    # substituted 'dynamically' in the environment for the corresponding
+    # service.
```

Comment:
Two small things:

1. Instead of just an example (e.g.) for default labels, can we document the list? Maybe a separate paragraph like:

```
The defaults labels for each protocol are:
- loki: pebble_service
```

2. Can you use a more specific word than 'dynamically' here? For example, "at the time the log is written" or "when the plan is updated" (I think it's the latter).

Reply:
I'd rather not document the list for a few reasons:
- it locks us in
- if it purports to be a definitive reference, someone will forget to update it eventually - then our docs will be lying
- no-one "should care" about what the default labels are - they are sensible defaults whose presence is not hurting anyone